src/Controller/Api/GalleryController.php

```diff
@@ -412,6 +398,8 @@ public function deleteGalleryAction($id)
     /**
      * Write a GalleryHasMedia, this method is used by both POST and PUT action methods.
      *
+     * @View(serializerGroups={"sonata_api_read"}, serializerEnableMaxDepthChecks=false)
```

Comment:
If this is the default value: `serializerEnableMaxDepthChecks=false` maybe we can remove it also?